3.evaluate_model/scripts/nbconverted/class_PR_curves.py

```diff
@@ -88,27 +83,135 @@
         compiled_class_PR_curves.append(PR_data)
 
 
-# ### Save PR curves from each evaluation
+# ### Save PR curves from each evaluation (multiclass)
 # 
 
 # In[4]:
 
 
 # compile list of tidy data into one dataframe
-compiled_class_PR_curves = pd.concat(compiled_class_PR_curves).reset_index(drop=True)
+# some thresholds are None because last PR value doesnt correspond to cell dataset (these values are always P=1, R=0), remove these rows from PR data
```

Comment:
```suggestion
# some thresholds are None because last PR value doesn't correspond to cell dataset (these values are always P=1, R=0), remove these rows from PR data
```

Reply:
Added to notebook in https://github.com/WayScience/phenotypic_profiling_model/pull/26/commits/987e1ec5a50e2751ae962ec53d06265164fcb301